### DISCUSSION TITLE
Afreecatv.py deosnt't work with Plugin options

### DISCUSSION THREAD

**Discussion by KOONWHITE** [streamlink/streamlink#5278] [Q&A] [answered]
### Description

Afreecatv.py

streamlink "https://play.afreecatv.com/a/b" best -o "c.ts" (working)

streamlink "https://play.afreecatv.com/a/b" --afreeca-username "d"  --afreeca-password "e" best -o "f.ts" (not working)
(ID and password have been entered correctly.)

The error 'streamlink: error: unrecognized arguments: best' is displayed.

What should I do?

**Comment by KOONWHITE** [comment]
i solved it
streamlink "https://play.afreecatv.com/a/b" best --afreeca-username "c" --afreeca-password "d" --afreeca-purge-credentials -o .ts
I pushed the value of 'best' forward and it works.

**Comment by bastimeyer** [comment, marked as answer]
The order of position and non-positional arguments can't be mixed.
https://streamlink.github.io/cli.html#positional-arguments

The only valid order is `executable [non-positional...] [positional...] [non-positional...]`
